Gruntfile.js

```diff
@@ -99,6 +111,12 @@ module.exports = function (grunt) {
         "mochaTest",
         "run:testRules",
     ].concat(checkBinTest));
+    grunt.registerTask("docs", [
```

Comment:
I understand what you're getting at: we want to make sure that `docs.ts` builds and passes the linter. However, we _don't_ want the part of the task to run that actually updates the docs site, because the docs site would advertise unreleased TSLint rules and options.

For simplicity for now, I'll adjust things so that the build part of the current docs task is run on every build, including CircleCI builds. The actual updating of the files in the tslint-gh-pages, that is, the execution of `buildDocs.js`, will still have to be done manually for the time being